### PR TITLE
LPS-117810 Deprecate SortedProperties

### PR DESCRIPTION
**Pull request by dantewang**

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portal/tools/TCKtoJUnitConverter.java [hunk 1]
```diff
@@ -135,7 +134,7 @@ private void _convert(String className, String message, File outputDir)
 		sb.append("\">\n");
 		sb.append("\t<properties>\n");
 
-		Properties properties = new SortedProperties(System.getProperties());
+		Properties properties = System.getProperties();
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/tools/TCKtoJUnitConverter.java:137]: @dantewang @ChiZelin I think we need order here. Without sorting, you will see the output is out of order.

Tina.

**Comment by tinatian** [comment]
@dantewang Please see my comment and resend.

Thanks.
Tina.

**Pull request closed without merging**